### PR TITLE
Use commitizen/conventional-commit-types

### LINKED ISSUE #271: Use commitizen/conventional-commit-types
Hello,

It would be great if this extension could use the same default types than the ones used by [commitizen/cz-conventional-changelog](https://github.com/commitizen/cz-conventional-changelog) for [commitizen/cz-cli](https://github.com/commitizen/cz-cli).

Reference : [commitizen/conventional-commit-types](https://github.com/commitizen/conventional-commit-types/blob/master/index.json)

https://github.com/KnisterPeter/vscode-commitizen/blob/61f6b887d3f1b183700599a916091e1958398b45/src/extension.ts#L141-L182

### PR DESCRIPTION
**Pull request by qkdreyer**
Fixes #271

### CODE REVIEW

**Comment by KnisterPeter** [comment]
@qkdreyer Awesome. Thanks for you contriubtion :rocket: 
I'll have to check whats wrong with travis... then I'll merge and release.

**Comment by KnisterPeter** [comment]
@qkdreyer Travis is is running again, can you please make sure the build succeeds?

**Comment by qkdreyer** [comment]
🙏

**Comment by KnisterPeter** [comment]
This `src/extension.ts:2:23 - error TS2307: Cannot find module 'conventional-commit-types'.` usally means there are no typescript types available for that module.

File: src/extension.ts [hunk 1]
```diff
@@ -4,6 +4,7 @@ import * as sander from 'sander';
 // tslint:disable-next-line:no-implicit-dependencies
 import * as vscode from 'vscode';
 import * as wrap from 'wrap-ansi';
+const types = require('conventional-commit-types').types;
```
Comment (KnisterPeter) [inline src/extension.ts]: I think its not a good idea to mix and match ESM and commonjs.
I would prefer if you use `import` syntax and provide typescript defintitions for the import if there are none available.

**Comment by KnisterPeter** [comment]
Closed, because no further feedback or progress.
Feel free to reopen/notify.

**Pull request closed without merging**